bson/raw_bson.py

```diff
@@ -13,6 +13,59 @@
 # limitations under the License.
 
 """Tools for representing raw BSON documents.
+
+Inserting and Retrieving RawBSONDocuments
+=========================================
+
+Example: Moving a document between different databases/collections
+
+.. testsetup::
+  from pymongo import MongoClient
+  client = MongoClient("localhost", 27017, document_class=RawBSONDocument)
+  client.drop_database('db')
+  client.drop_database('replica_db')
+
+.. doctest::
+
+  >>> import bson
+  >>> from pymongo import MongoClient
+  >>> from bson.raw_bson import RawBSONDocument
+  >>> client = MongoClient("localhost", 27017, document_class=RawBSONDocument)
+  >>> db = client.db
+  >>> result = db.test.insert_many([
+  ...  {'a': 1},
+  ...  {'b': 1},
+  ...  {'c': 1},
+  ...  {'d': 1}])
+  >>> assert result.acknowledged
```

Comment:
@ShaneHarvey is this necessary?

Reply:
Not needed. We can remove `result` entirely. I made a similar comment here: https://github.com/mongodb/mongo-python-driver/pull/578/files/1bea157142595b2f7b1472bb42035d958a3875ae#r596222833